docs/docker.rst

```diff
@@ -0,0 +1,84 @@
+.. inclusion-marker-start-do-not-remove
+
+Horovod in Docker
+=================
+
+To streamline the installation process on GPU machines, we have published the reference `Dockerfile <../Dockerfile>`__ so
```

Comment:
It does not work, good catch! However, I believe a custom anchor would not solve the issue, since `/Dockerfile` was not compiled with the documentation. I added the link like this `Dockerfile <https://github.com/horovod/horovod/blob/master/Dockerfile>`__ , which is supposed to work when it gets merged. Let me know if that works for you. If you any other suggestion as to how to have a link that points to both the Github repo and the compiled page inside the documentation, and that "knows" whether you are in the repo or docs to navigate you correctly, I would be glad.